app/services/idv/steps/recover_step.rb

```diff
@@ -1,7 +1,9 @@
 module Idv
   module Steps
     class RecoverStep < DocAuthBaseStep
-      def call; end
+      def call
+        create_document_capture_session(document_capture_session_uuid_key)
```

Comment:
I guess this should have always been here?

Reply:
It happens in the `WelcomeStep` during doc auth flow, so I think so?
https://github.com/18F/identity-idp/blob/master/app/services/idv/steps/welcome_step.rb#L6